content/reports/1000_IBM Power - Virtual Machines/report_test.yml

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
@CallisteH did you intend to include this `report_test.yml` file?

Reply:
@jaywcarman I had not seen that this file had remained. I removed it ! Thank you Jay